cannon/mipsevm/tests/fuzz_evm_multithreaded_test.go

```diff
@@ -15,8 +15,10 @@ import (
 )
 
 func FuzzStateSyscallCloneMT(f *testing.F) {
-	v := GetMultiThreadedTestCase(f)
+	versions := GetMultiThreadedTestCases(f)
+	require.NotZero(f, len(versions), "must have at least one multithreaded version supported")
 	f.Fuzz(func(t *testing.T, nextThreadId, stackPtr Word, seed int64) {
+		v := versions[len(versions)-1]
```

Comment:
Should we pick one of the versions randomly?

Reply:
Yeah that's a good idea - I was thinking that we'd wind up picking one to fuzz, but we actually pick a random one on each fuzz run so that works well.